test/unit/SupSub.test.js

```diff
@@ -158,10 +158,10 @@ suite('SupSub', function() {
       assert.equal(mq.latex(), 'x_a^b');
 
       mq.keystroke('Left Backspace');
-      assert.equal(mq.latex(), 'xb_a');
+      assert.equal(mq.latex(), 'x_ab');
```

Comment:
Sorry, I was misinterpreting what's happening here. `b` isn't being moved into the subscript, it's just being moved to being a sibling of x.

I guess that's fine? I don't really know what a user is trying to accomplish when they delete from the left end of an exponent that has other things in it.